alerter.c

```diff
@@ -1,20 +1,20 @@
-#include <stdio.h>
-#include <assert.h>
-
-int alertFailureCount = 0;
+#include "alerter.h"
 
 int networkAlertStub(float celcius) {
     printf("ALERT: Temperature is %.1f celcius.\n", celcius);
     // Return 200 for ok
     // Return 500 for not-ok
     // stub always succeeds and returns 200
-    return 200;
+    if(celcius <=MAX_STUB_VALUE)
+        return MAX_STUB_VALUE;
+    else
+        return THERSHOLD_STUB_VALUE;
 }
 
 void alertInCelcius(float farenheit) {
     float celcius = (farenheit - 32) * 5 / 9;
     int returnCode = networkAlertStub(celcius);

```

Comment:
The production code should not be aware of test code or test stub. It should be abstracted by the use of interface or function pointer.